src/01-functions.js

```diff
@@ -131,7 +147,13 @@ function compareTypes() {}
  *  flipSign(50);
  *  //> -50
  */
-function flipSign() {}
+ function flipSign(a) {
+  if (a){
+     return -a 
+  }else{
```

Comment:
none of this is incorrect here but if we wanted we really only need line 152 for this code to run.